src/pages/Xudts/index.tsx

```diff
@@ -37,29 +38,39 @@ const TokenInfo: FC<{ token: XUDT }> = ({ token }) => {
       name: t('xudt.address_count'),
       value: localeNumberString(token.addressesCount),
     },
+    {
+      name: t('xudt.created_time'),
+      value: token.createdAt ? parseDateNoTime(Number(token.createdAt) / 1000, false, '-') : null,
+    },
   ].filter(BooleanT())
 
   return (
-    <div key={token.typeHash} className={styles.tokenInfo}>
-      <div className={styles.title}>
-        {token.published ? (
-          <>
-            <Link className={styles.link} to={`/xudt/${token.typeHash}`}>
-              {symbol}
-            </Link>
-            {token.fullName ? <span>{token.fullName}</span> : null}
-          </>
-        ) : (
-          symbol
-        )}
-      </div>
+    <Card key={token.typeHash} className={styles.tokensCard}>
+      {token.published && (
+        <div className={styles.tokenInfo}>
+          {/* {token.fullName ? <span>{token.fullName}</span> : null} */}
+          <span className={styles.title}>Name</span>
+          <Link className={styles.link} to={`/xudt/${token.typeHash}`}>
+            {symbol}
+          </Link>
+        </div>
+      )}
+      {token.published && (
+        <div className={styles.tokenInfo}>
+          <span className={styles.title}>Symbol</span>
+          {token.fullName ? <span className={styles.value}>{token.fullName}</span> : null}
+        </div>
+      )}
       {fields.map(field => (
-        <dl key={field.name}>
-          <dt>{field.name}</dt>
-          <dd>{field.value}</dd>
-        </dl>
+        <div className={styles.tokenInfo}>
```

Comment:
Why use `div` instead of `dl/dt/dd`

Reply:
> Why use `div` instead of `dl/dt/dd`

dl, dt and dd has the global style which is designed in global.scss, and the styles are not compatible to the design draft, so I just use the div

Reply:
dd,dl, and dt are semantic,their style can be overridden.